mon-pix/app/components/data-protection-policy-information-banner.js

```diff
@@ -0,0 +1,32 @@
+import { inject as service } from '@ember/service';
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+
+export default class DataProtectionPolicyInformationBanner extends Component {
+  @service currentUser;
+  @service currentDomain;
+  @service intl;
+
+  get shouldDisplayDataProtectionPolicyInformation() {
+    const hasNotAlreadySeenDataProtectionPolicyInformation =
```

Comment:
@AndreiaPena si je mets ici  
console.log(' this.currentUser.user?', this.currentUser.user);
console.log(' this.currentUser', this.currentUser);

On s'aperçoit que le current user n'est pas chargé.
currentUser.user?.cgu === true > ça passe et ça donne false

Il faut recharger la page pour que le currentUser soit chargé  :-( 

![Capture d’écran 2023-01-02 à 11 01 09](https://user-images.githubusercontent.com/7688741/210216961-ae3b8bdd-af8c-4172-b60b-efaa60ab48bb.png)

Reply:
ps : j'ai la même chose avec un utilisateur nouvellement créé (j'ai testé avec George de Cambrigde plus haut)